pyproject.toml

```diff
@@ -1,6 +1,6 @@
 # LocalStack project configuration
 [build-system]
-requires = ['setuptools>=64,<=75.1.0', 'wheel', 'plux>=1.10', "setuptools_scm>=8.1"]
+requires = ['setuptools>=64', 'wheel', 'plux>=1.10', "setuptools_scm>=8.1"]
```

Comment:
Your changes say:
Bumps the minimum version of plux.

But this is not happening here, should we do a >=1.12?

Reply:
Thanks for the pointer, had the commit lying on my local copy without pushing it. 🤦🏽 Should be fixed in 8f75abf5fdf731bd3f35df5a7c059e5a9a6568a1.